JavaScript/d-obj-create.js

```diff
@@ -6,7 +6,7 @@ module.exports = creaters;
 creaters.arraySize = 1000; // Size of arrays in tests
 
 // Create array that consist of arraySize elements using mixins
-creaters.mixinCreate = function mixinCreate() {
+creaters.mixinCreate = () => {
```

Comment:
I feel like this change is appropriate until https://github.com/HowProgrammingWorks/Benchmark/pull/10 lands. This PR is likely to be merged earlier, so let it work with the current benchmarking harness.